### PR TITLE
Improve the issue template for dependency problems

### PR DESCRIPTION
**Pull request by willthames**
##### SUMMARY

So many reports of missing modules could be solved
more quickly with the info from `python_requirements_facts`

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
issue template

##### ADDITIONAL INFORMATION

See e.g. #43637

### CODE REVIEW

**Comment by willthames** [comment]
@gundalow I learned about the ISSUE_TEMPLATE directory the other day - should this change be moved there?

**Comment by gundalow** [comment]
Will: Maybe have it in all places where it could be useful (most apart from Docs template)?

FYI I pushed two commits to this branch. 

I didn't know about his module, so your education is working :)

**Comment by willthames** [comment]
I think it only makes sense in the bug report template, as that's when people are likely to have dependency issues (not for a feature request or even a security bug report)

**Review by gundalow — APPROVED** [review]

**Comment by willthames** [comment]
Merged based on conversation with @gundalow on IRC.

**Comment by dagwieers** [comment]
I am not happy with this change.
- We already established that multiline comments are a no-go because users add information in between comment-tags making the information not visible. That is why every line starts and ends with comment-tags for good reason.
- We used to have instructions telling people where they should add information, but half of the information in our template was instructions to users, rather than the bare basics.
- I also don't think we need to use the issue template for troubleshooting advice, because if that's the road we are taking, I have a lot more instructions for the user that are more important than dependency-related issues.

So I think we need to remove this, and keep the template short and concise as it was before. Only the bare necessities for a bug report. Not too much sections, and not too much information.

What we could do is add a single URL to a troubleshooting guide where we can go into detail about all sorts of issues and how to debug them, but again, I don't think that is the purpose of the issue template.

**Comment by dagwieers** [comment]
Removing the old ISSUE_TEMPLATE.md is also problematic, because some interface (i.e. mobile website) end up in the old ISSUE_TEMPLATE.md. So we cannot just remove it.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,51 +1 @@
...
-<!--- Describe what you expected to happen when running the steps above -->
-
-##### ACTUAL RESULTS
-<!--- Describe what actually happened. If possible run with extra verbosity (-vvvv) -->
-
-<!--- Paste verbatim command output between quotes -->
-```paste below
-
-```
+Please raise issues via the [new interface](https://github.com/ansible/ansible/issues/new/choose)
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE.md:1]: After this, all the comments made by ansibot now point to the wrong template.

cc @gundalow ^

**Pull request merged**